Source/WebCore/page/scrolling/ScrollingTreeNode.h

```diff
@@ -109,6 +114,7 @@ class ScrollingTreeNode : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr
 
     const ScrollingNodeType m_nodeType;
     const ScrollingNodeID m_nodeID;
+    FrameIdentifier m_parentFrameIdentifier;
```

Comment:
Can't this live on ScrollingTreeFrameScrollingNode?

Reply:
No we need a way to go from ScrollingNode -> FrameID for any node.